src/main/java/songbox/house/service/impl/UserPropertyServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package songbox.house.service.impl;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import songbox.house.domain.entity.user.UserInfo;
+import songbox.house.domain.entity.user.UserProperty;
+import songbox.house.repository.UserInfoRepository;
+import songbox.house.repository.UserPropertyRepository;
+import songbox.house.service.CurrentUserService;
+import songbox.house.service.UserPropertyService;
+import songbox.house.service.UserService;
+
+@Service
+@AllArgsConstructor
+public class UserPropertyServiceImpl implements UserPropertyService {
+    private final UserInfoRepository userInfoRepository;
+    private final UserPropertyRepository userPropertyRepository;
+    private final UserService userService;
+
+    @Override
+    public UserProperty getUserProperty() {
+        UserInfo currentUser = userService.getCurrentUser();
+        UserProperty userProperty = currentUser.getUserProperty();
+
+        if (currentUser.getUserProperty() == null) {
+            userProperty = new UserProperty();
+            currentUser.setUserProperty(userProperty);
+            userInfoRepository.save(currentUser);
```

Comment:
What's the reason of saving empty object?

Reply:
If there is no user property

Reply:
I asked the reason, not the condition

Reply:
> If there is no associated user property with current user
I hope it help you

Reply:
this saving to the database is pointless and takes memory